src/queue-list/lab-dialogs/add-to-worklist-dialog.component.tsx

```diff
@@ -279,30 +282,83 @@ const AddToWorklistDialog: React.FC<AddToWorklistDialogProps> = ({
                 />
               </div>
               {preferred && (
-                <div style={{ width: "500px" }}>
-                  <section className={styles.section}>
-                    <Select
-                      labelText={t("location", "Location ")}
-                      id="nextQueueLocation"
-                      name="nextQueueLocation"
-                      invalidText="Required"
-                      value={selectedNextQueueLocation}
-                      onChange={(event) =>
-                        setSelectedNextQueueLocation(event.target.value)
-                      }
-                    >
-                      {filteredlocations.map((location) => (
-                        <SelectItem
-                          key={location.uuid}
-                          text={location.display}
-                          value={location.uuid}
-                        >
-                          {location.display}
-                        </SelectItem>
-                      ))}
-                    </Select>
-                  </section>
-                </div>
+                <>
```

Comment:
weird. Is this extra enclosure necessary

Reply:
> weird. Is this extra enclosure necessary

not necessary will remove it